vendor/bat-native-ads/src/bat/ads/internal/ads_impl.h

```diff
@@ -21,13 +21,13 @@
 #include "bat/ads/ad_notification_info.h"
 #include "bat/ads/internal/ads_serve.h"
 #include "bat/ads/internal/bundle.h"
+#include "bat/ads/internal/classification/page_classifier/page_classifier.h"
 #include "bat/ads/internal/client.h"
 #include "bat/ads/internal/ad_conversions.h"
 #include "bat/ads/internal/ad_notification_result_type.h"
 #include "bat/ads/internal/ad_notifications.h"
 #include "bat/ads/internal/timer.h"
-#include "bat/ads/internal/page_classifier/page_classifier.h"
-#include "bat/ads/internal/purchase_intent/purchase_intent_classifier.h"
+#include "bat/ads/internal/classification/purchase_intent_classifier/purchase_intent_classifier.h"
```

Comment:
nit: include order

Reply:
Include order will be fixed with https://github.com/brave/brave-browser/issues/9166, thanks